shop.js

```diff
@@ -95,4 +97,27 @@ function showShoppingCartProducts() {
         totalPrice = totalPrice + shoppingCart[i].price;
     }
     document.getElementById('total').textContent = totalPrice.toFixed(2) + '€';
+};
+
+function getInvoice() {
+    const email = document.getElementById('emailInputField').value;
+
+    let body = 'Thank you very much for your registration at Krioukov Academy.<br>Here are the courses you are interested in: <ul>'
+    for (let i = 0; i < shoppingCart.length; i++) {
+        body = body + `<li>${shoppingCart[i].title} (${shoppingCart[i].dates}) - ${shoppingCart[i].price}€</li>`;
+    }
+
+    body = body + '</ul><br>We will contact you soon to clarify the last details. <br>Best regards, SBKG'
+    
+    Email.send({
```

Comment:
suggestion for improvements: since you want to use the same mechanism in two places you can create another js file (emailing.js or something) and put this function there. So you won't duplicate code and in case of changes will need to change it in one place.

Reply:
That's a good suggestion for future, thank you. 
As far as I am going to use this website as a prototype, let me leave it as it is